e2e/specs/date_input.spec.js

```diff
@@ -15,38 +15,36 @@
  * limitations under the License.
  */
 
-/// <reference types="cypress" />
-
 describe("st.date_input", () => {
   beforeEach(() => {
-    cy.visit("http://localhost:3000/");
-  });
+    cy.visit("http://localhost:3000/")
```

Comment:
This seems like a linter change, can we do it as a separate PR. I feel it makes the review much harder.

Reply:
Sure. This came from trying to merge with develop.